libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
This is actually a different bug; there's a missing `errno = t_save_errno` after the call to `unlink()` below.